.github/workflows/docker-publish.yaml

```diff
@@ -20,7 +20,11 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-
+    
+    - name: Set env
+      # github tag will be like refs/tags/v0.1.0 :10 removes the first 10 characters
+      run: echo RELEASE_VERSION=$(echo ${GITHUB_REF:10}) >> $GITHUB_ENV
+    
     - name: Login to Quay
```

Comment:
Can not we use `env` here itself

https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables

Reply:
AFAIK There is no standard env to get the part of GitHub tag.